templates/account/is-paying-disconnect-button.php

```diff
@@ -0,0 +1,56 @@
+<?php
+    /**
+     * @package     Freemius
+     * @copyright   Copyright (c) 2016, Freemius, Inc.
+     * @license     https://www.gnu.org/licenses/gpl-3.0.html GNU General Public License Version 3
+     * @since       2.5.0
+     */
+
+    if ( ! defined( 'ABSPATH' ) ) {
+        exit;
+    }
+
+    /**
+     * @var array $VARS
+     * @var Freemius $fs
+     */
+    $fs                 = $VARS['freemius'];
+    $premium_plan_title = $VARS['premium_plan_title'];
+    $slug               = $VARS['slug'];
+
+    $button_id = "fs-disconnect-button-{$fs->get_id()}";
+
+    $product_type = $fs->get_module_label( true );
+
+    /** translators: %1$s is replaced by the premium plan name, %2$s is replaced by the type of the module */
+    $message = $premium_plan_title ?
+        sprintf( fs_text_inline( 'Your %1$s plan is not active on this site at the moment and you can activate it on any other site. Only disconnect if you no longer want to use this %2$s anymore. If you wish to stop the recurring payment of the %1$s plan instead, then kindly click the CANCEL button, activate the %1$s plan and click on the "Downgrade" button to downgrade your account. Are you sure you would like to continue with this disconnection?', 'disconnect-x-confirm', $slug ), $premium_plan_title, $product_type ) :
+        sprintf( fs_text_inline( 'Only disconnect if you no longer want to use this %s anymore. Are you sure you would like to continue with the disconnection?', 'disconnect-confirm', $slug ), $product_type );
+
+?>
+<script type="text/javascript">
+    // Wrap in a IFFE to prevent leaking global variables.
+    (function() {
+        var disconnect_button_id = <?php echo wp_json_encode( $button_id ); ?>;
+        var message = <?php echo wp_json_encode( $message ); ?>;
+
+        jQuery(document).ready(function($) {
```

Comment:
@swashata WP JS formatting is add spacing after and before `'(', ')'`. Example:
```
$( '#' + disconnect_button_id ).click( function( e ) {
```

We only started to use that properly ~1yr ago, so you'll find old JS formatted differently. But I think it's good to follow WP practices going forward for the SDK.